From: Bardia Moshiri <fakeshell@bardia.tech>
--- a/src/hwcomposer.c	2023-06-12 09:38:46.100483192 +0000
+++ b/src/hwcomposer.c	2023-06-12 09:37:28.360972765 +0000
@@ -109,11 +109,8 @@ Bool hwc_hwcomposer_init(ScrnInfoPtr pSc
 	char device[10];
 	memset(device,0,sizeof(device));
 
-	read_file_into("/proc/device-tree/model", device, sizeof (device));
-	if (string_match(device, "MT6873")) {
         hwc->hwcVersion = HWC_DEVICE_API_VERSION_2_0;
 		return hwc_hwcomposer2_init(pScrn);
-	}
 
 	hwc_start_fake_surfaceflinger(pScrn);
 
